data/sql/updates/pending_db_world/rev_1691843684176706600.sql

```diff
@@ -0,0 +1,4 @@
+--
+DELETE FROM `spell_dbc` WHERE (`ID` = 42533);
```

Comment:
This is wrong, we dont fix spells in this table anymore. It is just for serverside spells not existing in the dbc files.

For any dbc corrections you do them in SpellInfoCorrections.cpp

Reply:
restore PlayerStorage.cpp file and move the spell_dbc stuff into SpellInfoCorrections.cpp and this should be okay